src/autoplex/auto/phonons/flows.py

```diff
@@ -266,6 +271,23 @@ def make(
         }
 
         for structure, mp_id in zip(structure_list, mp_ids):
+            self.supercell_settings.setdefault(mp_id, {})
+            supercell_matrix_job = reduce_supercell_size_job(
+                structure=structure,
+                min_length=self.supercell_settings.get("min_length", 15),
+                max_length=self.supercell_settings.get("max_length", 20),
+                fallback_min_length=self.supercell_settings.get(
+                    "fallback_min_length", 12
+                ),
+                max_atoms=self.supercell_settings.get("max_atoms", 500),
+                min_atoms=self.supercell_settings.get("min_atoms", 50),
+                step_size=self.supercell_settings.get("step_size", 1.0),
+            )
+            flows.append(supercell_matrix_job)
+            self.supercell_settings[mp_id][
+                "supercell_matrix"
+            ] = supercell_matrix_job.output
```

Comment:
@JaGeo , I restructured the code in a way that the `reduce_supercell_size_job` is only called once and the matrix is passed like that. But I have left the `reduce_supercell_size_job` (which only will be triggered if no matrix is gieven) in the other functions as it was before, too, in case a user wants to use the single-atom displ. and rattled supercell constructors separately.
I have thought a lot about it and I don't see a reason why single-atom displ. and rattled supercells of the same wf should have different settings. Do you think I shall still add the possibility?

Reply:
It might be that someone wants it. At the moment, I also do not see a clear use case. Let's, for now, keep it with this single job. Just make sure everything works :)

Reply:
I can see if I can maybe put some "NotImplemented" warning. In principle the user can still at the moment pass different settings if they call the supercell constructors separately.

Reply:
Added a simple user info instead